sysutils/lscpu/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                lscpu
+version             1.2.0
+revision            0
+github.setup        NanXiao ${name} ${version}
+github.tarball_from archive
+
+checksums           rmd160  b430777194dab4d99a028cc6e633b6b5507ddb09 \
+                    sha256  d91a87694bd14a26b1ed8cd44d62444d93e68eaf98db6488b3abd36ad09a6bcc \
+                    size    7863
+
+categories          sysutils
+maintainers         {none} openmaintainer
+license             BSD
+homepage            https://github.com/NanXiao/${name}
```

Comment:
`github.setup` will automatically set the `homepage`, you don't need to specify it explicitly.